docs/plugins/analysis-icu.asciidoc

```diff
@@ -21,6 +21,9 @@ sudo bin/elasticsearch-plugin install analysis-icu
 The plugin must be installed on every node in the cluster, and each node must
 be restarted after installation.
 
+This plugin can be downloaded for offline install from
```

Comment:
I would make `offline install` a link to https://www.elastic.co/guide/en/elasticsearch/plugins/current/installation.html#_custom_url_or_file_system - add an `[[id]]` to that section for easy linking.